tests/models/pix2struct/test_modeling_pix2struct.py

```diff
@@ -420,7 +420,11 @@ def prepare_config_and_inputs_for_common(self):
 class Pix2StructModelTest(ModelTesterMixin, PipelineTesterMixin, unittest.TestCase):
     all_model_classes = (Pix2StructForConditionalGeneration,) if is_torch_available() else ()
     all_generative_model_classes = (Pix2StructForConditionalGeneration,) if is_torch_available() else {}
-    pipeline_model_mapping = {"image-to-text": Pix2StructForConditionalGeneration} if is_torch_available() else {}
+    pipeline_model_mapping = (
+        {"image-to-text": Pix2StructForConditionalGeneration, "image-text-to-text": Pix2StructForConditionalGeneration}
+        if is_torch_available()
+        else {}
+    )
```

Comment:
these models are part of the pipeline mapping, can't we always have them set as pipeline model mapping by default in the ModelTesterMixin ?

Reply:
Technically it's possible to hide this in a background process (to infer which modeling classes of a model architecture for a task).

However when I reworked on the pipeline testing 2 years ago, we decided to make the things very explicit and easy to see the content, and that is why we ends up having this mapping.

Reply:
got it thanks